### DISCUSSION TITLE
"Too many failed login attempts" when configuring integration

### DISCUSSION THREAD

**Discussion by reidprichard** [SecKatie/ha-wyzeapi#446] [Q&A]
I've had this integration running on two different Home Assistant installations. Setting it up was initially a breeze in both cases. However, on one installation, the Wyze password was changed, forcing me to remove and re-add the integration. Over the past several days, trying a few times throughout each day, I've gotten the following error when trying to add the integration:  "Unable to login with response from Wyze: {'description': 'Too many failed attempts. Please try again later.', 'requestId': <removed>, 'errorCode': 1000}"

I know in the past I've gotten rate-limited _after_ setting up the integration due to all the initial requests for the state of every device, but it seems strange that just trying to log in a few times would hit a limit (if I recall, I was given the wrong password, so I did have a handful of failed login attempts before getting the right credentials). Anyone have an idea what's going on? Do I just need to wait a few days without trying to log in? Is it possible I've gotten my IP blacklisted? 

Edit: I'm wondering if I may still have the old credentials in my docker-wyze-bridge container, causing it to flood Wyze with bad login attempts. I'll update once I've checked that.

**Comment by JoeSchubert** [comment]
It may have. I have never actually looked at the rate limiting on the login endpoint.

However, I don't think that the password has any relevance for something that's already logged in. After a login things use the refresh token to revalidate their logins... so I'd be surprised if Wyzebridge would have caused bad login attempts if it had already obtained valid credentials.
